### ISSUE TITLE
perceptually uniform colormap for Heatmap

### ISSUE THREAD

**Issue by FelipeLema** [jupyter-widgets/ipyleaflet#416]
I call for an upgrade in colormap/gradient used in `Heatmap`. Several plotting libraries/packages have followed this trend of having viridis as default. (incl. Matplotlib, MATLAB and plotly)

More details of the reasons why [here](https://bids.github.io/colormap/).

**Comment by martinRenou** [comment]
I agree. I think at some point ipyleaflet will use bqplot's scales for the `Heatmap` and the `Choropleth` layers, which comes with a viridis colormap. And we could use the viridis one by default.
